### PR TITLE
 completed ticket 3 - created an end point for /api/topics

### PR DESCRIPTION
**Pull request by 779carney**

### CODE REVIEW

File: __tests__/get.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+describe('/api/topics', () => {
+    test('to GET an array of objects containing the following keys: slug, description', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.topics.length).toBe(3)
+                expect(response.body.topics[0].hasOwnProperty('slug')).toBe(true)
+
```
Comment (two-triangles) [inline __tests__/get.test.js:19]: add the same assertion for "description"

File: app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const {fetchTopics}= require("./controllers/controller.api");
+
+const app = express();
+app.use(express.json());
+
+app.get('/api/topics', fetchTopics);
+
+
+app.get('/*', (req,res)=>{
+res.status(404).send('not found')
```
Comment (two-triangles) [inline app.js:11]: should send back an object, rather than a string!

File: playground.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+\c nc_news
```
Comment (HannahBurks) [inline playground.sql:1]: Make sure to add this to your git ignore :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Please make the requested changes Elle and you should be good to go! Almost there :D keep it up

**Pull request merged**